test/test_unistd.rs

```diff
@@ -555,16 +555,13 @@ fn test_lseek() {
     const CONTENTS: &[u8] = b"abcdef123456";
     let mut tmp = tempfile().unwrap();
     tmp.write_all(CONTENTS).unwrap();
-    let tmpfd = tmp.into_raw_fd();
 
     let offset: off_t = 5;
-    lseek(tmpfd, offset, Whence::SeekSet).unwrap();
+    lseek(tmp.as_raw_fd(), offset, Whence::SeekSet).unwrap();
 
     let mut buf = [0u8; 7];
-    crate::read_exact(tmpfd, &mut buf);
+    crate::read_exact(&tmp, &mut buf);
     assert_eq!(b"f123456", &buf);
-
-    close(tmpfd).unwrap();
```

Comment:
I don't quite understand this `close(2)` operation, `tmpfd` will be closed when `tmp` (of type: `File`) dropped, why do we need this extra `close(2)`?

Reply:
I get it now, missed that 'into_raw_fd()' yesterday.

Will remove into_raw_fd and close.